packages/backend/src/services/UserAttributesService/UserAttributeUtils.ts

```diff
@@ -75,6 +75,9 @@ export const getFilteredExplore = (
         joinedTables: explore.joinedTables.filter((joinedTable) =>
             filteredTableNames.includes(joinedTable.table),
         ),
+        ...(filteredTableNames.length > 0
```

Comment:
no need to add this argument if there are no filtered tables, since this will be redundant